src/ngraph/codegen/compiler.cpp

```diff
@@ -472,8 +472,14 @@ void codegen::CompilerCore::configure_search_path()
     add_header_search_path(CUDNN_HEADER_PATHS);
 #endif
 
-#ifdef NGRAPH_DISTRIBUTED
+#ifdef NGRAPH_DISTRIBUTED_ENABLE
```

Comment:
Not needed? The same comment applies to the rest of the files.

Reply:
if that's done then you will throw an error when not in distributed mode